cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -889,6 +889,14 @@ spec:
             - name: KUBE_RBAC_PROXY_ENABLED
               value: {{ (quote .Values.global.prometheus.kubeRBACProxy) }}
             {{- end }}
+            {{- if .Values.csvCustomPricing }}
```

Comment:
```suggestion
            {{- if .Values.enterpriseCustomPricing }}
```

Reply:
Got it, thanks Jesse! Going to change all three instances in a forthcoming commit.